sdk/storage/storage-queue/src/Pipeline.ts

```diff
@@ -37,8 +37,8 @@ import { AnonymousCredential } from "./credentials/AnonymousCredential";
 export {
   BaseRequestPolicy,
   deserializationPolicy,
-  IHttpClient,
-  IHttpPipelineLogger,
+  HttpClient,
+  HttpPipelineLogger,
```

Comment:
Breaking change, @jeremymeng ?

Reply:
I believe @XiaoningLiu wanted to keep the `I-` for these two as they are intended for customer to implement so we didn't touch these two last time.

Reply:
Reverted